data-pipeline/Dockerfile

```diff
@@ -12,6 +18,7 @@ ARG USER_UID=2222
 ARG USER_GID=$USER_UID
 RUN groupadd --gid $USER_GID $USERNAME \
     && useradd --uid $USER_UID --gid $USER_GID -m $USERNAME
+RUN chown -R pipeline-user:pipeline-user /opt/
```

Comment:
Added this because running `pytest` and `ruff check` in the container throw errors when the user doesn't have access to write cache files.